docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -498,7 +488,6 @@ You may also refer to the [secrets-config proxy](../../security/secrets-config-p
     
     * `apps.secrets-config.proxy.tls.cert`
     * `apps.secrets-config.proxy.tls.key`
```

Comment:
@farshidtz 

I don't know how to code support for the pre-seeded cert.  I am pretty sure this is broken.

Reply:
There is wrapper around secrets-config which is implemented [here](https://github.com/canonical/edgex-snap-hooks/blob/main/options/secrets-config.go). It takes the snap options set by the user (cert and key payloads), writes them to a file, and calls secret-config internally. I have it in my to do list to move it to the edgex-go repo.

Yes it is broken. However, there is no way to make it work because secrets-config doesn't work right now in the snap because of the file ownership change: https://github.com/edgexfoundry/edgex-docs/pull/981#discussion_r1132273741